symbols/arm7.yml

```diff
@@ -3,55 +3,101 @@ arm7:
     - EU
     - NA
     - JP
+    - EU-WRAM
+    - NA-WRAM
+    - JP-WRAM
+    - EU-RAM
+    - NA-RAM
+    - JP-RAM
   address:
     EU: 0x2380000
     NA: 0x2380000
     JP: 0x2380000
+    EU-WRAM: 0x37F8000
+    NA-WRAM: 0x37F8000
+    JP-WRAM: 0x37F8000
+    EU-RAM: 0x27E0000
+    NA-RAM: 0x27E0000
+    JP-RAM: 0x27E0000
   length:
-    EU: 0x27080
+    EU: 0x27080    #0x1E8 long ENTRY block
```

Comment:
Sorry, `resymgen` (the YAML transpiler this repo relies on) doesn't support YAML comments. Let's just put this in the binary description block.

Reply:
Added